internal_transfers/actions/src/database.ts

```diff
@@ -104,6 +142,8 @@ export async function insertPipelineResults(
     await insertTokenImbalances(db, eventMeta.txHash, imbalances);
     await insertSettlementSimulations(db, settlementSimulations);
     await insertSettlementEvent(db, eventMeta, settlementEvent);
+    // TODO - markReceiptProcessed in follow up PR.
+    // await markReceiptProcessed(db, eventMeta.txHash);
```

Comment:
This change will happen in the next PR -- its not exactly related to the changes introduced here (i.e. just providing the table and operations on that table). This additional functionality is inter-twined with the business logic of the full project scope.

Reply:
Thanks for providing this additional comment here. Was confused for a bit.